README.md

```diff
@@ -52,6 +52,31 @@ in napalm.buildPackage ./. { packageLock = <path/to/package-lock>; }
 
 If you want to use Napalm in your flake project, you can do that by adding it to your inputs and passing `napalm.overlay` to your Nixpkgs instance. There is also a template, that can help you use napalm in your project. You can use it by running `nix flake init -t "github:nix-community/napalm"`.
 
+**Example**
+
+```nix
+{
+  inputs.url.nixpkgs = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  inputs.url.napalm = "github:nix-community/napalm";
+  
+  outputs = { self, nixpkgs, napalm }: 
+  let
+    system = "linux-x86_64";
+    pkgs = nixpkgs.legacyPackages."${system}";
+  in {
+    # Assuming the flake is in the same directory as package-lock.json
+    packages."${system}".package-name = napalm.legacyPackages."${system}".buildPackage ./. { };
+
+    devShells."${system}".shell-name = pkgs.mkShell {
+      nativeBuildInputs = with pkgs; [
+        nodejs
+        napalm-registry
```

Comment:
I am not sure why would one need the registry in the dev shell since they will likely have direct network access. And we do not document how to actually set it up anyway so if someone knows they need to use it, they can probably figure out how to install it.

Reply:
Listed that as I couldnt think of anything else to put in a devShell example. I'll remove it.

Reply:
Removed that one in 1e69d3b